### DISCUSSION TITLE
How to enable and start recording?

### DISCUSSION THREAD

**Discussion by sir-farfan** [Motion-Project/motionplus#113] [Q&A] [answered]
Hello
I just installed MotionPlus and added my one and only camera today: it's already showing video at 127.0.0.1:8080.

I mounted a new partition where I want to store the recordings 24/7 at /media/camera and created a new directory called outside with mod 777 and owner motion:motion

I tried adding target_dir to either and both of motionplus.conf and camera.conf
target_dir /media/camera/outside
But nothing gets saved there whether I'm watching the monitoring page or not. There are no error logs or warnings either.

What else do I need to do in order to enable and start recording?

I'm running Debian/Bookworm, installed using the provided .deb file.
I haven't configured the DB, not sure if it's needed for the recording to work.

**Comment by Mr-Dave** [comment, marked as answer]
1. Enable [emulate_motion](https://motion-project.github.io/motionplus_config.html#emulate_motion)
2. Enable [movie_output](https://motion-project.github.io/motionplus_config.html#movie_output)
3. Check the log to see if reports permission issues.

**Reply by sir-farfan** [reply]
Thanks, that worked.
